packages/orchestration/USAGE.md

```diff
@@ -0,0 +1,43 @@
+# Usage
+
+Last verified 2024-09-06
+
+This document describes some example contracts and test suites showing how to use Orchestration.
+
+## Example contracts
+
+See [`src/examples`](src/examples)
+
+
+| Contract Name | Status | Description | Features Used |
+|---------------|-------------|---------------|--------|
+| [auto-stake-it](/packages/orchestration/src/examples/auto-stake-it.contract.js) | Ready 🟢 | Sets up an IBC hook to automatically stake tokens on a remote chain received at a deposit address. | - `LocalOrchestrationAccount`<br>- `CosmosOrchestrationAccount`<br>- `Vtransfer` (IBC Hooks) |
+| [basic-flows](/packages/orchestration/src/examples/basic-flows.contract.js) | Ready 🟢 | Creates an account on a remote chain and returns a continuing offer with all platform-provided invitationMakers. | - `CosmosOrchestrationAccount`<br>- `LocalOrchestrationAccount`|
+| [query-flows](/packages/orchestration/src/fixtures/query-flows.contract.js) | Ready 🟢 | Test fixture that enables querying account balances on local and remote chains. | - `Chain`<br>- `LocalOrchestrationAccount`<br>- `CosmosOrchestrationAccount`<br>- Interchain Queries |
+| [sendAnywhere](/packages/orchestration/src/examples/sendAnywhere.contract.js) | Ready 🟢 | Allows sending payments (tokens) over IBC to another chain. | - `LocalOrchestrationAccoun`t<br>- `Vtransfer` (IBC Hooks) |
```

Comment:
we really should get consistent on the file naming. out of scope though.

I wish we had a tool that would fail CI if we refer to a file path in Markdown that doesn't exist. (so when we change the filenames this wouldn't get stale)

Reply:
We can tackle `stakeIca`/`stakeBld` with #9893. Having CI verify `sendAnywhere` here: https://github.com/Agoric/agoric-sdk/pull/10042